tools/rpmkeys.c

```diff
@@ -42,6 +41,20 @@ static struct poptOption optionsTable[] = {
     POPT_TABLEEND
 };
 
+static ARGV_t gpgkeyargs(ARGV_const_t args) {
+    ARGV_t gpgargs = argvNew();
+    for (char * const * arg = args; *arg; arg++) {
+	if (strncmp(*arg, "gpg-pubkey-", 11)) {
+	    char * gpgarg = rpmExpand("gpg-pubkey-", *arg, NULL);
```

Comment:
Again, why the macro expand? 'rpmkeys --delete %{somemacro}' seems like a weird use-case to support, I'd think for something like OpenPGP keys you'd rather always be explicit about the key.

Reply:
No idea why moving the code elsewhere didn't solve the issue...